xbmc/AppParamParser.h

```diff
@@ -25,19 +27,31 @@ class CAppParamParser
 
   const CFileItemList& GetPlaylist() const;
 
-  int m_logLevel;
-  bool m_startFullScreen = false;
-  bool m_platformDirectories = true;
-  bool m_testmode = false;
-  bool m_standAlone = false;
+  void SetLogLevel(int logLevel) { m_logTarget = logLevel; }
+  void SetStartFullScreen(bool startFullScreen) { m_startFullScreen = startFullScreen; }
+  void SetStandAlone(bool standAlone) { m_standAlone = standAlone; }
+
+  bool GetPlatformDirectories() const { return m_platformDirectories; }
```

Comment:
Should this be `HasPlatformDirectories()` ??

Reply:
yes